services/jupyter/src/pyproject.toml

```diff
@@ -26,6 +26,7 @@ transformers = "^4.22.2"
 typer = "^0.6.1"
 wandb = "^0.13.3"
 boto3 = {version = "^1.24.89", source = "pypi"}
+python-dotenv = "^0.21.0"
```

Comment:
Pydantic's `BaseConfig` in `app/config.py` parses environment variables automatically, see https://pydantic-docs.helpmanual.io/usage/settings/. These are currently being passed to the container through docker-compose.

Reply:
changed back